File: Backend/Java env/auth-service/Eclothing/src/main/java/com/example/demo/dto/UpdateSellerRequest.java
package com.example.demo.dto;

public class UpdateSellerRequest {
	private String name;
    private String mobile;

    private String shopName;
    private String gstNumber;
    private String businessAddress;
    
	public UpdateSellerRequest() {
	}

	public UpdateSellerRequest(String name, String mobile, String shopName, String gstNumber,
			String businessAddress) {
		super();
		this.name = name;
		this.mobile = mobile;
		this.shopName = shopName;
		this.gstNumber = gstNumber;
		this.businessAddress = businessAddress;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getMobile() {
		return mobile;
	}

	public void setMobile(String mobile) {
		this.mobile = mobile;
	}

	public String getShopName() {
		return shopName;
	}

	public void setShopName(String shopName) {
		this.shopName = shopName;
	}

	public String getGstNumber() {
		return gstNumber;
	}

	public void setGstNumber(String gstNumber) {
		this.gstNumber = gstNumber;
	}

	public String getBusinessAddress() {
		return businessAddress;
	}

	public void setBusinessAddress(String businessAddress) {
		this.businessAddress = businessAddress;
	}
}

File: Backend/DotNet env/AuthService/Models/DTOs/UpdateSellerRequest.cs
namespace AuthService.Models.DTOs
{
    public class UpdateSellerRequest
    {
        public string Name { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string Mobile { get; set; } = string.Empty;

        public string ShopName { get; set; } = string.Empty;
        public string GstNumber { get; set; } = string.Empty;
        public string BusinessAddress { get; set; } = string.Empty;
    }
}
